server/eclipse-project/src/main/java/us/freeandfair/corla/query/ContestQueries.java

```diff
@@ -109,10 +109,8 @@ public static Set<Contest> forCounty(final County the_county) {
   public static void deleteForCounty(final Long the_county_id) {
     final Set<Contest> contests = 
         forCounty(Persistence.getByID(the_county_id, County.class));
-    if (contests != null) {
```

Comment:
I really like the idea of going through everything and changing it from something/null with null checks to Optional(something or nothing) with .ispresent checks. I'm just a little unsure about this particular case - are you sure you changed forCounty so that it can never return null?

Reply:
You are completely right, it definitely can. I even have a test for it higher up! 🤦

Reply:
I have now put this back, and also changed the function to return a count of how many contests it deletes so transient DB errors can be detected.